java/runtime/src/main/resources/ray.default.conf

```diff
@@ -67,10 +71,6 @@ ray {
   // Redis configurations
   // ----------------------
   redis {
-    // The address of the redis server to connect, in format `ip:port`.
-    // If not provided, Ray processes will be started locally, including
-    // Redis server, Raylet and object store.
-    address: ""
     // If `redis.server` isn't provided, which port we should use to start redis server.
```

Comment:
Does other config key such as `head-password`/`password` still starts with `redis`?

Reply:
It doesn't matter because we don't document them.